src/runtime_src/core/edge/drm/zocl/include/zocl_util.h

```diff
@@ -138,20 +165,16 @@ struct drm_zocl_dev {
 	struct dma_chan		*zdev_dma_chan;
```

Comment:
soft_kernel and aie_information should also be part of drm_zocl_slot structure as these are also associated with the xclbin
Can you move soft_kernel and aie_information there?